cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/module/lib/inmemory/InMemoryTableModule.java

```diff
@@ -31,6 +30,5 @@ public void register(DatasetDefinitionRegistry registry) {
     registry.add(new InMemoryTableDefinition("table"));
     registry.add(new InMemoryTableDefinition(Table.class.getName()));
     registry.add(new InMemoryTableDefinition("orderedTable"));
```

Comment:
I saw your earlier comment. So that means that we will have to keep "orderedTable" for all times?